models/public/conv-deconvnet/model.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2020 Intel Corporation
```

Comment:
Why conv-deconvnet?

Reply:
because of just simple modification of original name - https://github.com/yalaudah/facies_classification_benchmark/blob/master/core/models/section_deconvnet.py

Reply:
I agree that it is not clear name, usually we provide mapping with original model name in index.md, so you can provide different from original name here, but make it more meaningful